setup.py

```diff
@@ -18,36 +18,40 @@
 from setuptools import setup
 
 
-PY3 = sys.version_info[0] == 3
-WINDOWS = sys.platform == 'win32' or sys.platform == 'cygwin'
+WINDOWS = sys.platform == "win32" or sys.platform == "cygwin"
 
-description = ("Thrift SASL Python module that implements SASL transports for "
-               "Thrift (`TSaslClientTransport`).")
+description = (
+    "Thrift SASL Python module that implements SASL transports for "
+    "Thrift (`TSaslClientTransport`)."
+)
 
 setup(
-    name='thrift_sasl',
-    version='0.4.2',
+    name="thrift_sasl",
+    version="0.4.2",
     description=description,
     long_description=description,
-    url='https://github.com/cloudera/thrift_sasl',
+    url="https://github.com/cloudera/thrift_sasl",
     install_requires=[
         # Python 3 support was added to thrift in version 0.10.0.
-        'thrift>=0.10.0' if PY3 else 'thrift==0.9.3',
+        "thrift>=0.10.0;python_version>='3.0'",
+        "thrift==0.9.3;python_version<'3.0'",
         # Installing sasl on Windows is rather painful, so use the pure python
         # implementation on Windows
-        'pure-sasl>=0.3.0' if WINDOWS else 'sasl>=0.2.1',
-        'six>=1.13.0'
+        "pure-sasl>=0.3.0;platform_system=='Windows'",
+        "sasl>=0.2.1;platform_system!='Windows'",
```

Comment:
Note, that in his latest comment @IMax153 suggested using 'sys_platform' instead of 'platform_system'.